src/_pytest/python.py

```diff
@@ -1428,15 +1428,13 @@ def write_fixture(fixture_def: fixtures.FixtureDef[object]) -> None:
         argname = fixture_def.argname
         if verbose <= 0 and argname.startswith("_"):
             return
-        if verbose > 0:
+        else:
             bestrel = get_best_relpath(fixture_def.func)
             funcargspec = f"{argname} -- {bestrel}"
```

Comment:
IMHO this gets just a tad more readable if you [eschew the extraneous else](https://www.youtube.com/watch?t=289&v=JVVMMULwR4s&feature=youtu.be) :wink:

```suggestion
        bestrel = get_best_relpath(fixture_def.func)
        funcargspec = f"{argname} -- {bestrel}"
```